sdk/signature-mpc-wasm/src/lib.rs

```diff
@@ -248,6 +248,14 @@ pub fn generate_keypair() -> Result<JsValue, JsErr> {
     Ok(serde_wasm_bindgen::to_value(&(public_key, private_key))?)
 }
 
+#[wasm_bindgen]
+pub fn generate_keypair_from_bytes(bytes: &[u8]) -> Result<JsValue, JsErr> {
+    let (public_key, private_key) =
+        signature_mpc::twopc_mpc_protocols::encrypt_user_share::generate_keypair_from_bytes(bytes)
```

Comment:
Here you should do the hash

Reply:
This would return a 32-byte array you can use as a seed

Reply:
Why it should be done here and not in the TS as it is being done now?